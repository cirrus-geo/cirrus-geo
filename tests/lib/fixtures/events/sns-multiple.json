{
  "Records": [
    {
      "Sns": {
        "Message": "{\"id\": \"some-collection/some-workflow/an-item-id1\"}"
      }
    },
    {
      "Sns": {
        "Message": "{\"id\": \"some-collection/some-workflow/an-item-id2\"}"
      }
    }
  ]
}
