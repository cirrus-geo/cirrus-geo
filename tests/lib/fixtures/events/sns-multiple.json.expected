[
    {
        "id": "some-collection/some-workflow/an-item-id1"
    },
    {
        "id": "some-collection/some-workflow/an-item-id2"
    }
]
